tests/nested/manual/core20-early-config/defaults.yaml

```diff
@@ -6,7 +6,7 @@ defaults:
       console-conf:
         disable: true
     watchdog:
-      runtime-timeout: 13m
+      runtime-timeout: 1m
```

Comment:
Thank you for making this smaller

Reply:
Thinking about this again I wonder a bit - the man page has no information why this might be too high and the qemu we create seems to not have any watchdog HW so I wonder if the error actually means that there is no watchdog HW? Or am I missing something?

Reply:
I think systemd has a limit on how long to wait for a watchdog - @sergiocazzolato mentioned that he was seeing some error log about this.

Reply:
Thanks, I looked a bit at this and it seems we get the `iTCO_wdt` watchdog (version 2) which comes from our usage of `-machine q35`.  According to the kernel docs that gives us `heartbeat: Watchdog heartbeat in seconds. (2<heartbeat<39 (TCO v1) or 613 (TCO v2), default=30` [1]. I think I would feel slightly better with a higher timeout (600s which should be fine) because our code often runs in slow nested machines so I'm worried that we might get occasional reboots because the timeout cannot be served in time.

[1] https://www.kernel.org/doc/html/latest/watchdog/watchdog-parameters.html

Reply:
I pushed a small commit (with proper context hopefully) that increases it and adds a check that the timeout can be set.